### DISCUSSION TITLE
16 bank messages

### DISCUSSION THREAD

**Discussion by deandonaldson** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#169] [Feature Requests & Ideas]
Can we get the 16 bank setting messages back please?

I have 10 midi devices on my board i want to patch change simultaneously + other things that right now i have to program another button to do the extra work after bank change. Id prefer that all to happen on patch/bank change.

Would need the 16 bank setting messages to achieve this please.

**Comment by simonaglover95** [comment]
Hi Dean, thanks for the info. You're the first person to mention that this has become an issue for them. 
I'll keep this issue open and we'll monitor the situation. Unfortunately we're limited in how many messages we can afford to have allocated to each setting at the moment.
